### ISSUE TITLE
Bot

### ISSUE THREAD

**Issue by horrill27** [eugenioclrc/binance-crypto-triangular-arbitrage#4]
can this be made into a bot that auto makes these trades once one is found?

**Comment by eugenioclrc** [comment]
Yes, you could port everything to node and use ccxt ( https://github.com/ccxt/ccxt ) to exec orders;
Example;
```js
exchange.createOrder(target.symbol, 'limit', 'buy', amount, target.ask),
exchange.createOrder(target.symbol, 'limit', 'sell', amount, target.bid)
```

**Comment by tgd1973** [comment]
Can this script send to a specific email address an email ... with orders to be done in the same way as tradingview send alert ?
I ask that there is a platform called GunBOT that knows how to work with most platforms via API and can also receive orders via email commands.
![Alert Tradingview](https://user-images.githubusercontent.com/15127860/144876601-57d4b33c-9835-4c7a-8424-61b448a7e628.png)

**Comment by kadirumutkara** [comment]
Hello Sir ;

How we can run this application on our local . Because we could not see detailed explanation on readme .Could you provide more detail on readme side.

**Comment by aknkaplanoglu** [comment]
Hi sir,
Where should we put api-key and secrets?
